.github/workflows/ci.yaml

```diff
@@ -139,6 +139,8 @@ jobs:
           echo "Done"
 
   test-refresh:
+    # skip refresh test temporarily because the changes are not compatible
+    if: false
```

Comment:
This disables the refresh tests temporarily, because these will fail now that the way we store password in the database has changed.

Reply:
Here's a ticket for that purpose: https://warthogs.atlassian.net/browse/DPE-7644